### ISSUE TITLE
Integration README needs to be updated for pre-commit

### ISSUE THREAD

**Issue by tangyfruits** [hadolint/hadolint#907]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
Following the INTEGRATION.md readme should not warn about "The 'rev' field of repo 'https://github.com/hadolint/hadolint' appears to be a mutable reference (moving tag / branch).  Mutable references are never updated after first install and are not supported."
### Actual behavior
Warning appears
### Steps to reproduce the behavior
Create a .pre-commit-config.yaml with contents
```
repos:
-   repo: https://github.com/hadolint/hadolint
    rev: v2.12.1-beta
    hooks:
      - id: hadolint
```
Then run
$ pre-commit run --all-files

$ pre-commit --version
pre-commit 2.20.0

I have a fork here that addresses this: https://github.com/tangyfruits/hadolint/blob/master/docs/INTEGRATION.md
